### PR TITLE
Add external stops support.

### PR DESCRIPTION
**Pull request by markwomack**

### CODE REVIEW

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino [hunk 1]
```diff
@@ -63,6 +68,13 @@ double motor_controller_cmd_timeout;
 double motor_left_speed;
 double motor_right_speed;
 
+// External stop state, false if no intervention required.
+bool external_stop;
```
Comment (griswaldbrooks) [inline Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: Initialize to `false`

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino [hunk 2]
```diff
@@ -63,6 +68,13 @@ double motor_controller_cmd_timeout;
...
+
+// Fob mode, 0 indicates no fobs should be checked. 1-4
+// indicates which fob to check.
+byte fob_mode;
```
Comment (griswaldbrooks) [inline Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: Initialize to `0`

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino [hunk 3]
```diff
@@ -101,7 +113,7 @@ RosParamHelper ros_param_helper(ros_nh);
 // to stop robot activity
 double cmd_vel_timeout_threshold;
 ros::Time last_cmd_vel_time;
-bool cmd_vel_timeout_stop = false; // Set to true if time out threshold has been exceeded
+bool cmd_vel_timeout_exceeded = false; // Set to true if time out threshold has been exceeded
```
Comment (griswaldbrooks) [inline Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: Initialize to `false`

**Review by griswaldbrooks — CHANGES_REQUESTED** [review]

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino (hunk 1 above)
Comment (markwomack) [reply Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: Done.

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino (hunk 2 above)
Comment (markwomack) [reply Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: Done.

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino (hunk 3 above)
Comment (markwomack) [reply Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: Already was. Updated comment though.

Comment (griswaldbrooks) [reply Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: This is an assignment to false, not an initialization. I know that for basic types there is no savings, but there is also no cost and it helps keep things consistent.

File: Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino [hunk 4]
```diff
@@ -63,6 +68,13 @@ double motor_controller_cmd_timeout;
 double motor_left_speed;
 double motor_right_speed;
 
+// External stop state, false if no intervention required.
+bool external_stop = false;
```
Comment (griswaldbrooks) [inline Arduino/victoria_teensy_ros_node/victoria_teensy_ros_node.ino]: This is an assignment. Initializations look like
`bool external_stop(false);`

**Review by griswaldbrooks — CHANGES_REQUESTED** [review]

**Comment by griswaldbrooks** [comment]
https://travis-ci.org/victoriarobotics/victoria_platform/jobs/221214382#L2097

You should rebase.

**Comment by markwomack** [comment]
Rebase complete.  Atlassian FTW.

**Pull request merged**